ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/Claims.scala

```diff
@@ -46,119 +46,126 @@ final case class ClaimActAsParty(name: Ref.Party) extends Claim
-
-  def validForParticipant(id: String): Either[AuthorizationError, Unit] =
-    Either.cond(
-      participantId.forall(_ == id),
-      (),
-      AuthorizationError.InvalidParticipant(participantId.get, id),
-    )
+sealed trait ClaimSet
+
+object ClaimSet {
+  object Unauthenticated extends ClaimSet
+
+  /** [[Claims]] define what actions an authenticated user can perform on the Ledger API.
+    *
+    * They also optionally specify an expiration epoch time that statically specifies the
+    * time on or after which the token will no longer be considered valid by the Ledger API.
+    *
+    * The following is a full list of services and the corresponding required claims:
+    * +-------------------------------------+----------------------------+------------------------------------------+
+    * | Ledger API service                  | Method                     | Access with                              |
+    * +-------------------------------------+----------------------------+------------------------------------------+
+    * | LedgerIdentityService               | GetLedgerIdentity          | isPublic                                 |
+    * | ActiveContractsService              | GetActiveContracts         | for each requested party p: canReadAs(p) |
+    * | CommandSubmissionService            | Submit                     | for submitting party p: canActAs(p)      |
+    * | CommandCompletionService            | CompletionEnd              | isPublic                                 |
+    * | CommandCompletionService            | CompletionStream           | for each requested party p: canReadAs(p) |
+    * | CommandService                      | *                          | for submitting party p: canActAs(p)      |
+    * | LedgerConfigurationService          | GetLedgerConfiguration     | isPublic                                 |
+    * | PackageService                      | *                          | isPublic                                 |
+    * | PackageManagementService            | *                          | isAdmin                                  |
+    * | PartyManagementService              | *                          | isAdmin                                  |
+    * | ResetService                        | *                          | isAdmin                                  |
+    * | TimeService                         | GetTime                    | isPublic                                 |
+    * | TimeService                         | SetTime                    | isAdmin                                  |
+    * | TransactionService                  | LedgerEnd                  | isPublic                                 |
+    * | TransactionService                  | *                          | for each requested party p: canReadAs(p) |
+    * +-------------------------------------+----------------------------+------------------------------------------+
```

Comment:
Why not add the `HealthService` to this list so later developers don't forget about it?

Reply:
Good catch! I'll add the missing items to the list.

Reply:
Fixed.